macOS/DuckDuckGo/NavigationBar/View/AddressBarTextField.swift

```diff
@@ -67,8 +67,8 @@ final class AddressBarTextField: NSTextField {
 
     weak var onboardingDelegate: OnboardingAddressBarReporting?
     weak var focusDelegate: AddressBarTextFieldFocusDelegate?
-
-    private let searchPreferences: SearchPreferences = SearchPreferences.shared
+    weak var searchPreferences: SearchPreferences?
+    weak var tabsPreferences: TabsPreferences?
```

Comment:
AddressBarTextField is instantiated from a storyboard so instead of passing preferences objects in the initializer I’m using weak vars here and setting them in viewDidLoad of the view controller.